modules/ROOT/pages/release-notes/release-notes55.adoc

```diff
@@ -107,31 +108,31 @@ tinymce.init({
 
 For information on using:
 
-* The `image_list` option, see: link:{baseurl}/plugins/opensource/image/#image_list[Image plugin - `image_list`].
-* The `image_class_list` option, see: link:{baseurl}/plugins/opensource/image/#image_class_list[Image plugin - `image_class_list`].
-* The `link_class_list` option, see: link:{baseurl}/plugins/opensource/link/#link_class_list[Link plugin - `link_class_list`].
-* The `link_list` option, see: link:{baseurl}/plugins/opensource/link/#link_list[Link plugin - `link_list`].
-* The `table_class_list` option, see: link:{baseurl}/plugins/opensource/table/#table_class_list[Table plugin - `table_class_list`].
-* The `table_cell_class_list` option, see: link:{baseurl}/plugins/opensource/table/#table_cell_class_list[Table plugin - `table_cell_class_list`].
-* The `table_row_class_list` option, see: link:{baseurl}/plugins/opensource/table/#table_row_class_list[Table plugin - `table_row_class_list`].
+* The `image_list` option, see: xref:plugins/opensource/image.adoc#image_list[Image plugin - `image_list`].
+* The `image_class_list` option, see: xref:plugins/opensource/image.adoc#image_class_list[Image plugin - `image_class_list`].
+* The `link_class_list` option, see: xref:plugins/opensource/link.adoc#link_class_list[Link plugin - `link_class_list`].
+* The `link_list` option, see: xref:plugins/opensource/link.adoc#link_list[Link plugin - `link_list`].
+* The `table_class_list` option, see: xref:plugins/opensource/table.adoc#table_class_list[Table plugin - `table_class_list`].
+* The `table_cell_class_list` option, see: xref:plugins/opensource/table.adoc#table_cell_class_list[Table plugin - `table_cell_class_list`].
```

Comment:
This xref doesn't resolve as the `table_cell_class_list` id does exist on the heading in `table_cell_class_list.adoc`. e.g:
```
[[table_cell_class_list]]
=== `table_cell_class_list`
```

Reply:
done